charms/tensorboards-web-app/src/charm.py

```diff
@@ -130,6 +139,31 @@ def _configure_mesh(self, interfaces):
                 }
             )
 
+    def _on_sidebar_relation_joined(self, event: RelationJoinedEvent):
+        if not self.unit.is_leader():
+            return
+        event.relation.data[self.app].update(
+            {
+                "config": json.dumps(
+                    [
+                        {
+                            "app": self.app.name,
+                            "position": 4,
```

Comment:
I wonder what happens with the position if this is the only existing element in the sidebar. Do you know?

Reply:
nothing it will be first and the order of items ion the array will be preserved